### PR TITLE
feat: add sort income

### PR DESCRIPTION
**Pull request by alonakaliuzhna**
https://user-images.githubusercontent.com/83176493/185414082-56ad27bb-f470-4272-a12e-29715ffc2ec1.mp4

### Description

adding sort for income page, **the amount is commented out until the filter is ready!!**

### Type of change

- [x] Feature (introduces new functionality)
- [ ] Bugfix
- [ ] Configuration
- [ ] Refactoring (doesn't change functionality, only the code)
- [x] Style changes (doesn't affect functionality, only the look of the components)

### PR Checklist

- [x] Synced up with latest from the main branch/Code is linted
- [x] Description of the changes has been added (Screenshots/Video)
- [x] Chakra components/constants/hooks have been used wherever possible

> If not explain here

- [ ] Commit messages follow [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)
- [x] Console has been checked for warnings

#### Screenshots/Video before change

Add screenshots from the [current deployed state](https://sandbox-money-app.herokuapp.com/) of the changed components here.

#### Screenshots/Video after change

Add screenshots with the changes applied here.

### CODE REVIEW

File: src/pages/Expenses.jsx [hunk 1]
```diff
@@ -162,26 +160,6 @@ export const Expenses = () => {
...
       </Flex>
-      <Flex justify="flex-start" w="100%" h="30px">
-        <Flex
-          align="center"
-          ml="2%"
-          cursor="pointer"
-          onClick={() => onSetSortDate()}
-        >
-          <IconButton icon={<ArrowUpDownIcon />} variant="unstyled" />
-          <Text> {i18next.t('pageExpenses.sort.byDate')}</Text>
```
Comment (Yulatun) [inline src/pages/Expenses.jsx:173]: you should delete it from JSON file too

Comment (alonakaliuzhna) [reply src/pages/Expenses.jsx:173]: done

**Review by Yulatun — APPROVED** [review]

File: src/api/Transaction.js [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
+  pageParam = 1
+) => {
+  const queryString = new URLSearchParams({
+    PageNumber: pageParam,
+    IsSortByDate: false,
+    IsSortByAmount: false,
+    IsSortDescending,
+    [sortColumn]: false
+  }).toString();
+  return instance.get(`/api/v1/Transaction/Income?${queryString}`);
```
Comment (itsnotpolza) [inline src/api/Transaction.js:16]:
```suggestion
  return instance.get(`/api/v1/Transaction/Income`, { params: { PageNumber, ....} });
```

Comment (alonakaliuzhna) [reply src/api/Transaction.js]: done

**Review by itsnotpolza — APPROVED** [review]

**Review by lazzzaroni — APPROVED** [review]

**Pull request merged**